app/helpers/image_encode_helper.rb

```diff
@@ -0,0 +1,33 @@
+module ImageEncodeHelper
+  def encodable_image_tag(source, options = {})
+    image_tag(encodable_image_source(source), options)
+  end
+  alias eimage_tag encodable_image_tag
+
+  def encodable_image_source(source)
+    if base64_encode_images? && source.present?
+      base64_encoded_uri(source)
+    else
+      path_to_image(source)
+    end
+  end
+  alias eimage_source encodable_image_source
+
+  def base64_encode_images?
+    @base64_encode_images
+  end
+
+  def base64_encoded_uri(source)
+    asset = Rails.application.assets[source]
+
+    if asset.content_type == 'image/svg+xml'
```

Comment:
Do we actually need a special SVG branch here?

Guess what, base64 encoded images are smaller :) - for our SVGs, the size sum is 1215 kB URL-encoded and 1189 kB base64 encoded.

So, simpler code *and* smaller images...